test/sagemaker_tests/tensorflow/tensorflow2_training/resources/smmodelparallel/smdataparallel_smmodelparallel_mnist_script_mode.sh

```diff
@@ -4,5 +4,6 @@
 
 set -ex
 
-smddpsinglenode python smdataparallel_mnist.py
+smddp_singlenode_dev python smdataparallel_mnist.py
```

Comment:
I am not sure what the first argument does and why have we changed it. Can you please explain?

Reply:
SMDDP team suggested to make such change. My understanding is smddp launcher of single node is only for development purpose. @roywei may have more context here.